### PR TITLE
feat: add `withoutNew` option

### PR DESCRIPTION
**Pull request by satazor**
Also fix a bug where the proto inheritance chain was not setup properly.
Refactored tests to their own files and improved the overall test suite.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,26 +109,38 @@ In such cases you can use the `withIs.proto` method:
...
+}, {
+    className: 'Circle',
+    symbolName: '@org/package/Circle',
+});
 ```
 
-The example above will print:
-```
-true
+...or event better:
```
Comment (fsdiogo) [inline README.md]: ...or `even` better:

File: README.md [hunk 2]
```diff
@@ -140,11 +160,18 @@ Type: `String`
 
 Unique *id* for the class. This should be namespaced so different classes from different modules do not collide and give false positives.
 
-Example: `@organization/package/class`
+Example: `@organization/package/Class`
+
+### withIs.proto(Class, { className, symbolName, withoutNew })
+
+The `className` and `symbolName` parameters are the save as above.
```
Comment (fsdiogo) [inline README.md]: are the `same`as above.

Comment (satazor) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (satazor) [reply README.md]: Fixed.

**Review by fsdiogo — APPROVED** [review]
Great job 👍

**Comment by satazor** [comment]
Thanks @fsdiogo 

You may merge and release!

**Pull request merged**